modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -1136,6 +1136,14 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		IndexWriterHelperUtil.updateDocument("test123", 123, null, null);
 	}
 
+	public void method(
+		String searchEngineId, long companyId, Document documents,
+		boolean commitImmediately) {
+
+		IndexWriterHelper.updateDocument(searchEngineId, companyId, documents, commitImmediately);
+		IndexWriterHelper.updateDocument("test123", 123, null, null);
```

Comment:
@MicaelleSilvaa you have added the reference at the bottom to use it to call the method, isn't a static method, so should use the variable.